fe/projects/AspNetCore/src/areas/Products/components/ProductDetail/ProductDetail.js

```diff
@@ -112,123 +117,183 @@ function ProductDetail(props) {
+                                            <ResponsiveImage sources={image.sources} imageSrc={image.src} imageAlt={image.alt} />
+                                        </LazyLoad>
+                                    )
+                                })}
+                            </div>
+                            {props.images && props.images.length > 6 &&
+                                <div className="product-detail__more-product-images">
+                                    {showMoreImages ? (
+                                        <div className="is-flex is-justify-content-center">
+                                            <Button variant="outlined"
+                                                onClick={handleShowMoreImages}
+                                                color="inherit">
+                                                {props.readLessText}
+                                            </Button>
+                                        </div>
+                                    ) : (
+                                        <div className="is-flex is-justify-content-center">
+                                            <Button variant="outlined"                                                
+                                                onClick={handleShowMoreImages}
+                                                color="inherit">
+                                                {props.readMoreText}
+                                            </Button>
+                                        </div>
+                                    )}
                                 </div>
-                            )}
+                            }
                         </div>
-                    }
-                    {props.description &&
-                        <div className="product-detail__product-description">
-                            <h3 className="product-detail__feature-title">{props.descriptionLabel}</h3>
-                            <div dangerouslySetInnerHTML={{__html: marked.parse(props.description)}}></div>
+                        <div className="column product-detail__mobile-gallery">
+                            {props.images && props.images.length > 0 &&
+                                <Splide
+                                    options={{
+                                        type: "slide"
+                                    }}
+                                >
+                                    {props.images.map((image, index) => {
+                                        return (
+                                            <SplideSlide key={index}>
+                                                <LazyLoad offset={LazyLoadConstants.defaultOffset()} className="product-detail__mobile-gallery__mobile-product-image">
+                                                    <ResponsiveImage sources={image.sources} imageSrc={image.src} imageAlt={image.alt} />
```

Comment:
**suggestion:** paramete imageSrc is wrong passed. Should be image.imageSrc instead of image.src